kamon-metrics/src/main/scala/com/ovoenergy/effect/Metrics.scala

```diff
@@ -24,9 +25,15 @@ object Metrics {
   /**
    * An instance of metrics for a Sync[F] which wraps the underlying calls
    */
-  def syncKamonMetrics[F[_]: Sync: FlatMap]: Metrics[F] =
-    metric =>
+  implicit def SyncMetrics[F[_]: Sync: FlatMap]: Metrics[F] = new Metrics[F] {
+    override def counter(metric: Metric): F[Long => F[Unit]] =
       FlatMap[F].map(Sync[F].delay(Kamon.metrics.counter(metric.name, metric.tags)))(
         counter => times => Sync[F].delay(counter.increment(times))
-    )
+      )
+
+    override def histogram(metric: Metric): F[Long => F[Unit]] =
+      FlatMap[F].map(Sync[F].delay(Kamon.metrics.histogram(metric.name, metric.tags)))(
```

Comment:
lgtm but maybe import `cats.syntax.functor._` for a more normal `map` experience

Reply:
👍but how come you were using `FlatMap`?